index.js

```diff
@@ -1,232 +1,403 @@
-const tierPoints = { 0: 0, 1: 1, 2: 2, 3: 5, 4: 10 };
-let totalPoints = 0;
-let form;
-
-const disclaimerConfirmButton = document.querySelector(
-  ".disclaimerConfirmButton"
-);
-disclaimerConfirmButton.addEventListener("click", () => {
-  document.querySelector(".disclaimerScreen").remove();
-});
-
-(function () {
-  //adds tier tags
-  const buttons = document.querySelectorAll(
-    ".levels .combinedOffencesContainer .offenceContainer button"
-  );
-  buttons.forEach((button) => {
-    const tag = document.createElement("span");
-    tag.classList.add("tier-tag");
-    button.parentNode.insertBefore(tag, button.nextSibling);
-  });
-})();
+const offences = {
+  //"offenceName": offenceTier
+  dogpiling: 0,
+  "discussing windows": 0,
+  "chat bombing": 0,
+  "discussing mental health issues": 0,
+  "inappropriate profile": 0,
 
-//if you go with the tier level I suggest the below order for Tier 0 to 10:
-const svgPaths = {
-  0: "./images/zaps/TOPzap-shade-1.svg", //tier 0  #ffdc2f
-  1: "./images/zaps/TOPzap-shade-2.svg", //tier 1  #eeb434
-  2: "./images/zaps/TOPzap-shade-3.svg", //tier 2  #e09034
-  3: "./images/zaps/TOPzap-shade-4.svg", //tier 5  #d47032
-  4: "./images/zaps/TOPzap-shade-5.svg", //tier 10 #be1e2d
-};
+  "mini-modding": 1,
+  "requesting out of scope help": 1,
+  "mild unprofessionalism": 1,
 
-const colorCodes = {
-  0: "#ffdc2f",
-  1: "#eeb434",
-  2: "#e09034",
-  3: "#d47032",
-  4: "#be1e2d",
+  "discussing piracy": 2,
+  "unsolicited pings/dms": 2,
+  "discussing politics/religion": 2,
+  "mild toxicity": 2,
+  "self promoting without permission": 2,
+
+  "discussing illegal activities": 3,
+  "arguing over moderation": 3,
+  "excessive toxicity": 3,
+
+  bigotry: 4,
+  "continued harrassment": 4,
+  "nsfw or highly offensive content": 4,
+  spamming: 4,
+  doxxing: 4,
 };
```

Comment:
We should probably be using a sort of nested object structure here to contain all the information we need about each tier and the offenses. Consider if we would want to add another tier? there's a lot of places in the code that need to be adjusted to fit that. Instead one could use this single object to describe the matrix, and write code around it to describe the behavior.

I would probably use an array of objects like so:

```js
[
  {
    tier: 0,
    zaps: 0,
    offenses: ["dogpiling", "discussing windows", "etc.."]
   },
   {
     tier: 1,
     zaps: 1,
     offenses: ["list", "of", "offenses"],
    },
    {
      tier: 2,
      zaps: 3,
      offenses: ["example", "example", "etc"]
    }
    // etc..
]
```

This allows us to be very explicit about what each part does when we write code that uses this object as a sort of source of truth and it makes it very easy to be descriptive in our code. We do not need to have any sort of prerequisite knowledge of the zaps and it helps make our code more generic.